.eslintignore

```diff
@@ -1,2 +1,4 @@
 frontend/src/cljs
 frontend/src/cljs_release
+e2e/support/cypress_sample_database.js
+e2e/support/cypress_sample_instance_data.js
```

Comment:
Both of these files import an automatically generated JSON file which is not tracked in git.

In CI ESLint reports `import/no-unresolved` violation, so we need to `// eslint-disable import/no-unresolved` it.
But locally the JSON files exist, so ESLint reports `Unused eslint-disable directive (no problems were reported from 'import/no-unresolved')`.

To make ends meet, the simplest solution was to to just ignore these files.

Reply:
sounds reasonable